integrationTests/json/scenariosExecutorLogs_test.go

```diff
@@ -242,6 +290,10 @@ func TestEgldEsdtSwapLog(t *testing.T) {
 }
 
 func TestPingPongEgldLog(t *testing.T) {
+	if !testexecutor.IsWasmer1Allowed() {
```

Comment:
why are you skipping these things ?

Reply:
Wasmer 2 can conflict with Wasmer 1 during initialization. These tests are intended to initialize both Wasmer 1 and Wasmer 2, which leads to a SIGABRT panic if there was a Wasmer 2 executor already initialized.

The function `IsWasmer1Allowed()` only returns true if the default executor is Wasmer 1, which ensures there is no previous Wasmer 2 executor initialized. This means the tests can be safely executed.